### PR TITLE
Admin, behind feature toggle `admin_style_v2`: new navigation menu

### LINKED ISSUE #9998: Changing back office main menu colors
## Description

**- As a:** enterprise user or super admin
**- On page:** Main menu in the backend: **this work will need to be behind a feature toggle**
**- I want to be able to do:** 

One the feature toggle is activated, I want to see a main menu in orange instead of blue (color `F27052` ): 

When selected, a lighter orange must be in place : 

As a first iteration : let's perhaps try to have the submenu with the same orange colors so we can introduce the fact that pressing an item in the submenu changes the colors to lighter orange as well (currently we don't have a signal in place when pressing submenu).

Turn text of both main and submenu with these size rules: 

Title of the page must be turned to black, with this size : 

No background color anymore for the title of the page.

Don't know if we want to go further, buut while we are at the top, let's turn this text black as well : 

## Acceptance Criteria & Tests

At this stage, perhaps only product will test, without circling the QA team. We will follow normal process, once we are sure of the colors implementation.

1. Log in as admin with feature activated
2. See the main menu colors as changed as described above

If login with feature toggle activated, old menu should still be in place.

### PR DESCRIPTION
**Pull request by jibees**
#### What? Why?

- Closes #9998

This introduce a brand new orange navigation menu, accessible via feature toggle `admin_style_v2 `:
Capture d’écran 2022-11-18 à 09 55 18

#### What should we test?

- As an admin, _with_ feature `admin_style_v2` activated, you should see the design
- As an admin, _without_ feature `admin_style_v2`, nothing should change

#### Release notes

Changelog Category: User facing changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Comment by jibees** [comment]
Should we wait for code review before _testing_ it?
What about deploy the PR in staging and make some feedback @RachL ?

**Comment by RachL** [comment]
@jibees I'm afraid of maiking decision based on the menu only. I think we need to do a full page before iterating more? As it's against a feature toggle, we can merge easily.

**Comment by jibees** [comment]
I can't do (or not easy) the full page before this PR is merged. That's what, i proposed you to deploy and test in staging and then you can merge (if it's ok to merge without code review).

What do you think?

**Comment by RachL** [comment]
> I can't do (or not easy) the full page before this PR is merged.

Yes I know. But I don't want to make you change things now. Let's merge after code review. I can check if everything is working as before during the test (after code review), but I'm not sure it's needed. 
What I'm saying is that I can't test if the color is ok. I think it is biased to do it now.

File: app/helpers/spree/admin/navigation_helper.rb
Comment (dacook) [inline app/helpers/spree/admin/navigation_helper.rb:37]: I'd like to suggest a way DRY this format which is shared with `link_to_with_icon`, but can't think of a way... 🤔

File: app/webpacker/css/admin/components/navigation.scss
Comment (dacook) [inline app/webpacker/css/admin/components/navigation.scss:80]: Except you accidentally left the indent spaces ;)
```suggestion
      text-align: center;
```

File: app/webpacker/css/admin/v2/colors.scss
Comment (dacook) [inline app/webpacker/css/admin/v2/colors.scss:7]: I like that you've included 'v2' to avoid conflicts. This is then effectively a namespace, so I think it would be better to have this at the start of the name (eg `$v2-orange`).

**Review by dacook — CHANGES_REQUESTED** [review]
Looks nice, although personally I find it a bit too much orange :P  
I have some questions and suggestions.

Do you think that the `v2` folder would remain separate, or would it eventually get merged up one level? 
In that case, it might be better to mirror the file structure, eg rename `v2/colors.scss` to `v2/variables.scss`. 
And it would be helpful to add a code comment inside to show the intention.

File: app/webpacker/css/admin/components/navigation.scss
Comment (jibees) [reply app/webpacker/css/admin/components/navigation.scss]: Thanks ;)

File: app/webpacker/css/admin/v2/colors.scss
Comment (jibees) [reply app/webpacker/css/admin/v2/colors.scss:7]: Yep, I agree, will do that.

**Review by dacook — APPROVED** [review]

**Comment by RachL** [comment]
I will stage this quickly so we can move forward: for the release notes, it would be cool to not mention this to instance managers yet. I will properly communicate when colors are ready to be seen (available on more pages)

**Comment by RachL** [comment]
the feature toggle works, so merging!

**Pull request merged**
